### ISSUE TITLE
Create Niflots that represent a fraction of a receiver's income

### ISSUE THREAD

**Issue by elmariachi111** [elmariachi111/niflot#5]
Receiver receives $5000/m by origin
Receiver can mint a niflot with a percentage based flowrate, eg 50% of the origin stream
Receiver can mint another niflot (25%) on the same stream until all niflots on that stream are adding up to 100% of the incoming flowrate.
Receiver can sell those niflots to several investors.

**Comment by joeljesu** [comment]
Yep, this is valuable. I believe the blocker here was just internal flow rate accounting and we were looking to produce this as a niflot v2.
